server/svix-server/src/wait_for.rs

```diff
@@ -1,43 +1,38 @@
 use std::time::Duration;
 
+const ATTEMPT_TIMEOUT: Duration = Duration::from_secs(3);
```

Comment:
Small nit - but I actually prefer where the constants were before, since they're only used in a single function, I think it makes sense to keep them restricted to that scope.